source/wireversion-featurelist.rst

```diff
@@ -0,0 +1,54 @@
+====================================
+Server Wire version and Feature List
+====================================
+
+.. list-table::
+   :header-rows: 1
+
+   * - Server version
+     - Wire version
+     - Feature List
+
+   * - 2.4 and before
+     - 0
+     - Text search
+
+   * - 2.6
+     - 1
+     - | Aggregation cursor
+       | Auth commands
+
+   * - 2.6 
+     - 2
+     - | Write commands (insert/update/delete)
+       | Aggregation $out pipeline operator
+  
+   * - 3.0
+     - 3
+     - | List collections
+       | List indexes
+       | SCRAM-SHA1
+       | explain command 
+
+   * - 3.2
+     - 4
+     - | (find/getMore/killCursors) commands
```

Comment:
currentOp and fsyncUnlock commands as well. Basically, anything that was previously implemented as a query on a special virtual collection became a command.

Reply:
Done.